c172-checklists.xml

```diff
@@ -2721,9 +2736,15 @@
                 <scale>3.1983</scale>
             </marker>
             <condition>
-                <property>sim/model/c172p/securing/pitot-cover-visible</property>
+                <or>
+                    <property>sim/model/c172p/securing/pitot-cover-visible</property>
+                    <not><property>/sim/model/c172p/securing/allow-securing-aircraft</property></not>
```

Comment:
This should be the opposite, right? We want this item to be considered only if securing is allowed AND if the pitot tube is NOT visible yet. Also I think the `<not>` looks better if it's in a different line.

Reply:
No - this makes sure that the checklist is green whenever securing is false - otherwise you have all this yellow / amber text in the checklist that you can't set to green as securing is disabled. 

As for the <not> I can change it

Reply:
I see, I think you are right!